assets/src/edit-story/app/media/pagination/reducer.js

```diff
@@ -122,6 +124,26 @@ function reducer(state = INITIAL_STATE, { type, payload }) {
       };
     }
 
+    case media3pTypes.MEDIA3P_SET_SEARCH_TERM: {
```

Comment:
As per offline discussion, this should be moved to media3p/providerReducer.js rather than depending on pagination/reducer.

Currently this won't affect local state but since pagination/reducer is used for both local state and media3p provider states, having this here makes it unclear that it only affects media3p state.

Reply:
Done.